keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -753,6 +798,21 @@ when drafting this test plan.
 
 ### Graduation Criteria
 
+#### Alpha -> Beta Graduation
+
+- A detailed DNS spec for multi-cluster services.
+- NetworkPolicy either solved or explicitly ruled out.
+- API group chosen and approved.
+- Kube-proxy can consume ServiceImport and EndpointSlice.
+- E2E tests exist for MCS services.
+- Beta -> GA Graduation criteria defined.
+- At least one MCS DNS implementation
+
+#### Beta -> GA Graduation
+
+- Scalability/performance testing, understanding impact on cluster-local service
```

Comment:
Do we expect to add additional features before GA?

Reply:
I believe we have a useful feature set as is to go GA. There is some related work in development - Service Topology, Gateway, Dual Stack, and the idea of a safer IP reservation system (Tim mention at the sig-mc weekly). As any/all of those progress we'll want to make sure they are incorporated or at least carefully considered.